code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -228,6 +225,13 @@
 
 /obj/machinery/particle_accelerator/control_box/ui_interact(mob/user)
 	. = ..()
+
+	if(construction_state != PA_CONSTRUCTION_COMPLETE)

```

Comment:
This in the wrong order? The `PA_CONSTRUCTION_PANEL_OPEN` condition will never trip.

Reply:
It seems to work perfectly. Tested it thoroughly.

Reply:
does clicking with an empty hand while the state is CONSTRUCTION_PANEL_OPEN bring up wires? should it?

Reply:
Actually, no it doesn't I confused myself. I think it should too, just incase.